third_party/move/move-prover/src/lib.rs

```diff
@@ -53,12 +53,12 @@ pub fn run_move_prover<W: WriteColor>(
             paths: options.move_sources.clone(),
             named_address_map: addrs.clone(),
         }],
-        vec![],
         vec![PackagePaths {
             name: None,
             paths: options.move_deps.clone(),
             named_address_map: addrs,
         }],
+        vec![],
```

Comment:
I am wondering why we need to switch move_deps and deps here, which seems to be inconsistent with the code here: https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/tools/move-package/src/compilation/compiled_package.rs#L762?

Reply:
Ok, I'm changing it back in this PR, since it's tangential, anyway.

Reply:
I've also filed https://github.com/aptos-labs/aptos-core/issues/13392 with some further analysis/discussion on the issue.  Not sure there's anything to fix there, but something to think about.